seqr/views/apis/dataset_api.py

```diff
@@ -1,3 +1,6 @@
+from __future__ import unicode_literals
+from builtins import str
```

Comment:
`from builtins import str` is necessary when there are unicodes in the object standard Python 2 `str` will fail to convert the object to a string (`unicode` is required).